react/week2/exersice-react/src/components-todo/TodoList.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react'
+import TodoItems from './TodoItems'
+import TodoListItem from './TodoListItem'
+
+function TodoList() {
+    const originalTodos = TodoItems();
+    let [updatedTodos, setTodos] = useState(originalTodos);
+
+    let newTodoDescription = '';
+
+    function addNewTodo() {
+        const id = updatedTodos.length + 1;
+        const date = new Date().toLocaleDateString("en-US", { weekday: 'long', year: 'numeric', month: 'long', day: 'numeric' });
+        const newTodo = { id: id, description: newTodoDescription, date: date }
+        updatedTodos = updatedTodos.concat(newTodo);
+        setTodos(updatedTodos)
+    }
+
+    return (<div>
+        <input type='text' onChange={e => newTodoDescription = e.target.value} />
+        <button className='add-button' onClick={() => addNewTodo()}>Add todo</button>
+        <ul>
+            {updatedTodos.map(todo => <TodoListItem key={todo.id} id={todo.id} description={todo.description} date={todo.date} />)}
```

Comment:
I believe it would be better to send todo itself by props then choose what you want to use in TodoListItem component. In addition to it, we have to send removeTodo function as we have delete button in the TodoListItem component.
```suggestion
            {todos.map((todo) => <TodoListItem key={todo.id} todo={todo} removeTodo={removeTodo} />)}
```